api/src/test/java/org/openmrs/module/registrationcore/api/RegistrationCoreSensitiveTestBase.java

```diff
@@ -1,9 +1,10 @@
 package org.openmrs.module.registrationcore.api;
 
+import org.junit.Ignore;
 import org.openmrs.test.BaseModuleContextSensitiveTest;
 import org.springframework.test.context.ContextConfiguration;
 
+@Ignore

```

Comment:
This is not the right thing to do, to me this looks like it was meant to be a base class and should be marked as abstract rather than adding @Ignore

Reply:
Changed to abstract and removed @ignore, I added it after https://talk.openmrs.org/t/registrationcore-mdule-tests-fail-even-in-a-fresh-copy/12833/7?

Reply:
Thanks! By our code convention this class would be named BaseRegistrationCoreSensitiveTest instead.